libzecale/circuits/fields/fp12_2over3over2_gadgets.tcc

```diff
@@ -197,110 +199,110 @@ Fp12_2over3over2_mul_by_024_gadget<Fp12T>::Fp12_2over3over2_mul_by_024_gadget(
-          FMT(annotation_prefix, " _z5_x4"))
-    , _z1_x0(
+          FMT(annotation_prefix, " _compute_z5_x4"))
+    , _compute_z1_x0(
           pb,
           Z._c0._c1,
           X_0,
-          result._c0._c1 +
-              ((_z2_x2.result + _z5_x4.result) * -Fp6T::non_residue),
-          FMT(annotation_prefix, " _z1_x0"))
+          result._c0._c1 + ((_compute_z2_x2.result + _compute_z5_x4.result) *
+                            -Fp6T::non_residue),
+          FMT(annotation_prefix, " _compute_z1_x0"))
     // z0*x2 + z2*x0 = (z0 + z2)*(x0 + x2) - z0*x0 - z2*x2
     // out_z2 = z0*x2 + z2*x0 + z3*x4
     //        = (z0 + z2)*(x0 + x2) - z0*x0 - z2*x2 + z3*x4
     // => (z0 + z2)*(x0 + x2) = out_z2 + z0*x0 + z2*x2 - z3*x4
-    , _z3_x4(
+    , _compute_z3_x4(
           pb,
           Z._c1._c0,
           X_4,
           libsnark::Fp2_variable<Fp2T>(pb, FMT(annotation_prefix, " z3*x4")),
-          FMT(annotation_prefix, " _z3_x4"))
-    , _z02_x02(
+          FMT(annotation_prefix, " _compute_z3_x4"))
+    , _compute_z02_x02(
           pb,
           Z._c0._c0 + Z._c0._c2,
           X_0 + X_2,
-          result._c0._c2 + _z0_x0.result + _z2_x2.result +
-              (_z3_x4.result * -FieldT::one()),
-          FMT(annotation_prefix, " _z02_x02"))
+          result._c0._c2 + _compute_z0_x0.result + _compute_z2_x2.result +
+              (_compute_z3_x4.result * -FieldT::one()),
+          FMT(annotation_prefix, " _compute_z02_x02"))
     // z2*x4 + z4*x2 = (z2 + z4)*(x2 + x4) - z2*x2 - z4*x4
     // out_z3 = z3*x0 + non_residue * (z2*x4 + z4*x2)
     //        = z3*x0 + non_residue * ((z2 + z4)*(x2 + x4) - z2*x2 - z4*x4)
     // => (z2 + z4)*(x2 + x4) = (out_z3 - z3*x0) / non_residue + z2*x2 + z4_x4
-    , _z3_x0(
+    , _compute_z3_x0(
           pb,
           Z._c1._c0,
           X_0,
           libsnark::Fp2_variable<Fp2T>(pb, FMT(annotation_prefix, " z3*x0")),
-          FMT(annotation_prefix, " _z3_x0"))
-    , _z24_x24(
+          FMT(annotation_prefix, " _compute_z3_x0"))
+    , _compute_z24_x24(
           pb,
           Z._c0._c2 + Z._c1._c1,
           X_2 + X_4,
-          _z2_x2.result + _z4_x4.result +
-              (result._c1._c0 + _z3_x0.result * -FieldT::one()) *
+          _compute_z2_x2.result + _compute_z4_x4.result +
+              (result._c1._c0 + _compute_z3_x0.result * -FieldT::one()) *
                   Fp6T::non_residue.inverse(),
-          FMT(annotation_prefix, " _z24_x24"))
+          FMT(annotation_prefix, " _compute_z24_x24"))
     // z0*x4 + z4*x0 = (z0 + z4)*(x0 + x4) - z0*x0 - z4*x4
     // out_z4 = z0*x4 + z4*x0 + non_residue * z5*x2
     //        = (z0 + z4)*(x0 + x4) - z0*x0 - z4*x4 + non_residue * z5*x2
     // => (z0 + z4)*(x0 + x4) = out_z4 + z0*x0 + z4*x4 - non_residue * z5*x2
-    , _z5_x2(
+    , _compute_z5_x2(
           pb,
           Z._c1._c2,
           X_2,
           libsnark::Fp2_variable<Fp2T>(pb, FMT(annotation_prefix, " z5*x2")),
```

Comment:
If the gadget is `compute_z5_x2` (or `compute_z5_times_x2`), I'd call the output wire `z5_x2` (or `z5_times_x2`) then

Reply:
changed all of these